<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>API Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>API:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="api">
            <item>
             <property name="text">
              <string notr="true">OpenGL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Vulkan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Null</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Device:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="device"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="use_disk_shader_cache">
          <property name="text">
           <string>Use disk shader cache</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_asynchronous_gpu_emulation">
          <property name="text">
           <string>Use asynchronous GPU emulation</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Internal Resolution:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="resolution_factor_combobox">
            <item>
             <property name="text">
              <string>Auto (Window Size)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Native (1280x720)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x Native (2560x1440)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3x Native (3840x2160)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x Native (5120x2880)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="ar_label">
            <property name="text">
             <string>Aspect Ratio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="aspect_ratio_combobox">
            <item>
             <property name="text">
              <string>Default (16:9)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Force 4:3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Force 21:9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stretch to Window</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="bg_label">
            <property name="text">
             <string>Background Color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bg_button">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
